packages/framework/src/shared/actions.ts

```diff
@@ -40,6 +40,22 @@ export type NavigateModalScreenAction =
 
 export type NavigateScreenAction = string | NavigateScreenOptions;
 
+interface ShowDialogOptions {
+  minWidth?: number;
+  maxWidth?: number;
+  minHeight?: number;
+  maxHeight?: number;
+  horizontalOffset?: number;
+  verticalOffset?: number;
+  style?: "default" | "none";
```

Comment:
I think we will need some more options here for web to control positioning.

Reply:
can you please enlist some, thanks.

Reply:
Something like the modal screen, being able to position to left/right/top/bottom/center